examples/README.md

```diff
@@ -28,7 +28,12 @@ For example: `dotnet run -- --server https://my_server admin "my password"`.
 Examples
 --------
 
-### Configuration
+### Common
 
 - [configuration](configuration): Demonstrates the different ways to
   supply a configuration for SystemLink APIs.
+
+### [Message](message)
+
+- [Send and Receive](message/send_receive): Demonstrates how to use the
+  SystemLink Message client API to send and receive messages between sessions
```

Comment:
We may want to consider capitalizing "Client" throughout the doc simply because we're including it in the API name. If you disagree, just let me know. :)